hugo/content/020_getting-started/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Getting Started
+url: /about/
+remote: https://github.com/gardener/gardener/blob/master/README.md
```

Comment:
the link under "To start using or developing the Gardener locally" isn't rendered as markdown but as raw markdown source

Reply:
78d4c73e70b6825996bbec93814c36b94a09baf8